### ISSUE TITLE
social media cards sprint 2

### ISSUE THREAD

**Issue by jdkunesh** [Chicago/design-cds-bootstrap#61]
- [x] avatar
- [x] shared content on the platform
- [x] shared image template (ie- shared a web page on this platform)
- [x] default shared image: what shows up if you don't use a shared image template

1 for each.

**Comment by jdkunesh** [comment]
https://twitter.com/ChiDesignDir/status/1214662353586835456

**Comment by alaeder16** [comment]
@jdkunesh ,

I added avatar and shared image template to Figma.
https://www.figma.com/file/t1oXrlay6I8WiDnz58vnu8/Chicago-Design-System-Digital-sticker-sheet-v0.9?node-id=8406%3A2575

As I can see cityofchicago doesn't have one official twitter or any other social media accounts. I would suggest to create one. Current Mayor's account is not official - it is her personal, so it needs to roll under the city's official account. We definitely need to have a page that provides all the links ( NY example) https://www1.nyc.gov/connect/social-media.page

I did some research on other cities and we are the weakest in that area.
